hack/convert-output-to-tsv.py

```diff
@@ -1,175 +1,157 @@
-import json
+#!/usr/bin/env python
+
+import argparse
 import csv
-from os import listdir
-import sys
+import json
+import logging
 from math import ceil
+import os
 
-# CLI args
-if (len(sys.argv) < 3):
-    printd("Please provide the input directory and output directory as arguments!")
-input_dir = sys.argv[1]
-output_dir = sys.argv[2]
-# optional debug arg, pass any string as the third arg
-try:
-    debug = sys.argv[3]
-except IndexError:
-    debug = None
-
-table_names = ['cslb', 'hles_cancer_condition', 'hles_dog', 'hles_health_condition', 'hles_owner', 'environment']
-pk_prefix = 'entity:'
-
-# debug printer
-def printd(x):
-    if debug:
-        print("[DEBUG] "+x)
-    else:
-        pass
+from gcsfs.core import GCSFileSystem
+
+
+def remove_prefix(text, prefix):
+    if text.startswith(prefix):
+        return text[len(prefix):]
+
+    return text
+
+
+parser = argparse.ArgumentParser(description="Convert the records in a specified bucket prefix to a Terra-compatible TSV")
+parser.add_argument('input_dir', metavar='I', help='The bucket prefix to read records from')
+parser.add_argument('output_dir', metavar='O', help='The local directory to write the resulting TSVs to')
+parser.add_argument('table', nargs='*', help="One or more tables to process into TSVs. Processes all tables if none specified.")
+parser.add_argument('--debug', action='store_true', help="Write additional logs for debugging")
+parser.parse_args()
```

Comment:
🔧 I think you need to assign the parsing result to a var here, i.e., `args = parser.parse_args()`, and then use that var when pulling out the args like `args.table`, `args.input_dir`, etc.

Reply:
yup, you're right. not sure how i missed that.